virtualcluster/pkg/syncer/resources/pod/dws.go

```diff
@@ -64,6 +64,14 @@ func (c *controller) Reconcile(request reconciler.Request) (res reconciler.Resul
 	if err := c.MultiClusterController.Get(request.ClusterName, request.Namespace, request.Name, vPod); err != nil && !apierrors.IsNotFound(err) {
 		return reconciler.Result{Requeue: true}, err
 	}
+	if featuregate.DefaultFeatureGate.Enabled(featuregate.TenantScheduler) {
```

Comment:
let's move `pPod, err := c.podLister.Pods(targetNamespace).Get(request.Name)` after this check to skip unneeded superCluster pod lookup?

Reply:
ok